src/ngraph/runtime/cpu/pass/cpu_fusion_bprop.cpp

```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************
```

Comment:
ngraph/runtime/cpu/pass/workspace_insertion.cpp? Probably ask around if folks have better name.

Reply:
*Loud thudding sounds of Nick's head banging against a keyboard*